### PR TITLE
chore: Check websocket message type in all providers

### PR DESCRIPTION
**Pull request by rbajollari**
## Description

closes: #9 

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [ ] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] added appropriate labels to the PR
- [ ] targeted the correct branch
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items._

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

File: oracle/provider/binance.go [hunk 1]
```diff
@@ -239,7 +239,11 @@ func (p *BinanceProvider) getCandlePrices(key string) ([]types.CandlePrice, erro
 	return candleList, nil
 }
 
-func (p *BinanceProvider) messageReceived(_ int, bz []byte) {
+func (p *BinanceProvider) messageReceived(messageType int, bz []byte) {
+	if messageType != websocket.TextMessage {
```
Comment (adamewozniak) [inline oracle/provider/binance.go:243]: Is the expected message type for all these providers `websocket.TextMessage`? iirc a lot of these don't follow the expected set of rules

Comment (rbajollari) [reply oracle/provider/binance.go:243]: This is how this check was set before we got rid of it. I'll test out all the providers though before merging to make sure it still works.

Comment (adamewozniak) [reply oracle/provider/binance.go:243]: 👍 thank you, sounds good

**Review by adamewozniak — APPROVED** [review]
pre-approving ❤️

**Pull request merged**